components/shared/IndexIssueListItem.js

```diff
@@ -23,7 +23,6 @@ export default function IndexIsssueListItem({ type, issue }) {
   const [description, img] = parseBody(issue.body);
   return (
     <Link
-      className="text-primary text-decoration-none"
```

Comment:
i changed the default styling of links in the css file to make them all default to this so we don't have to write it everywhere

Reply:
Your update makes link styling more consistent which I fully support. I think this is a great improvement. 👏  

Previously, links in some places underlined on hover while others did not - like the links in the paragraph below the "Our Work " header on `/projects?status=in_progress`. Now, these behave like the others.